test/helper.js

```diff
@@ -0,0 +1,34 @@
+let Application = require('spectron').Application
```

Comment:
let's call this file `modifySettings`or `modifySettingsHelper` ? `helper` is not very descriptive